test/instrumentation/decode_metric.go

```diff
@@ -219,6 +257,37 @@ func decodeBuckets(kv *ast.KeyValueExpr) ([]float64, error) {
 	return buckets, nil
 }
 
+func decodeBucketArguments(fc *ast.CallExpr) (float64, float64, int, error) {
```

Comment:
What are the return args supposed to represent?

Reply:
Decode arguments for prometheus.LinearBuckets and prometheus.ExponentialBuckets(float64, float64, int)